src/style/global.css

```diff
@@ -0,0 +1,61 @@
+/* Layout */
+.container-0 {
+  @apply mx-auto px-2 sm:px-4 lg:px-8;
+}
+
+.page-frame {
+  @apply container-0 overflow-x-hidden mt-4 mb-10;
```

Comment:
Let's make this w-10/12 so all pages have a standard width.

Reply:
dicussed